### PR TITLE
TRPL: Add "verboten" to the glossary

### PR DESCRIPTION
**Pull request by tomjakubowski**
Maybe a more suitable resolution to the issues raised in #19929 and #20439.

### CODE REVIEW

**Comment by killercup** [comment]
I'd expect the glossary to only contain explanations of technical terms specific to the book's topic (i.e., computer science, programming languages, etc.).

So, if I did not know _verboten_ is German for _forbidden_, I'd not look for it there. (I'd probably just use a regular dictionary.)

**Comment by tshepang** [comment]
Yes to @killercup, this does not belong here. Rather change all instances of that word to *prohibited*.

**Comment by bluss** [comment]
I think it's fine to keep using that word. You don't need it in the glossary, you can look it up in a regular dictionary if you want. I'm guessing there will be other words in the book that some english speakers won't be familiar with and will look up as well.

File: src/doc/trpl/glossary.md [hunk 1]
```diff
@@ -63,3 +63,9 @@ themselves form part of larger expressions.
 
 In computer programming, a statement is the smallest standalone element of a
 programming language that commands a computer to perform an action.
+
+### Verboten
+
+*adjective*
+
+1. forbidden, as by law; prohibited.
```
Comment (bluss) [inline src/doc/trpl/glossary.md]: If we would add this, it should be a quote with sources, since it's a verbatim copy from dictionary.com's definition.

**Comment by steveklabnik** [comment]
> So, if I did not know verboten is German for forbidden, 

It is also English for forbidden.

The glossary is intended, as @killercup mentions, as a way to lead people through computer-science specific education that they may not have received.

> I'm guessing there will be other words in the book that some english speakers won't be familiar with and will look up as well.

Yes, this is true for all text.

Thanks for the suggestion, @tomjakubowski , but this isn't a solution, nor do I think we need one, specifically.

**Pull request closed without merging**